.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.7.x
+  - 1.9.x
```

Comment:
I see 1.8.3 in most other APID components. If you depend on golang 1.9.x then should this be updated for rest of APID components?